lib/dradis/plugins/projects/clone.rb

```diff
@@ -0,0 +1,39 @@
+module Dradis::Plugins::Projects
+  class Clone
```

Comment:
```suggestion
  class CloneService
```

as this file as a service makes more sense

Reply:
Also, I don't think this class should be in the plugin too

Reply:
Got it, will move to the main app.